### ISSUE TITLE
Improve toolbar icon visibility for dark themes

### ISSUE THREAD

**Issue by smaragdus** [alct/export-tabs-urls#13]
The tool bar icon of **Export Tabs URLs** is hardly visible on darker background (when a theme like [this](https://addons.mozilla.org/en-US/firefox/addon/dark-gold/) is installed), screen:

![Export Tabs URLs 0 2 8 - 2019-06-03 - 005](https://user-images.githubusercontent.com/9025757/58833318-6ff43f00-8659-11e9-90a5-edd7bee8520d.png)

It is a bit more discernible when the mouse pointer hovers over the icon, screen:

![Export Tabs URLs 0 2 8 - 2019-06-03 - 006](https://user-images.githubusercontent.com/9025757/58833393-99ad6600-8659-11e9-834b-507183a90de6.png)

A possible solution might be an option to use an alternative icon (inverted colours or yellow background in stead of white background).

As a workaround I have placed it in between two lighter tool bar icons so that I can see the icon easier.

**Comment by alct** [comment]
Hello,

Thank you for your feedback!

This is a known issue. I have been waiting to redesign the ETU logo in order to properly fix it.

In the meantime, I have pushed a temporary fix in 0.2.9, see https://github.com/alct/export-tabs-urls/commit/e08301f85474764115069822e40d94e062d290ab (you will notice that even if the icon is now visible, it probably doesn't look good, this will be fixed with the logo redesign)

**Comment by smaragdus** [comment]
@alct 

Thanks for your response. This is a minor visual issue and I have no problem using the extension as I remember its location in the tool-bar. Anyway, it will look better if it is theme-aware.

I find your extension extremely useful- it saves me lots of manual copying, good work, thank you.

**Comment by smaragdus** [comment]
After the extension was updated it looks better on the dark theme I am using, thanks.

I wonder- would it be possible that **Export Tabs URLs** gets the colour from the currently used theme? The extension [Undo Close Tab](https://github.com/M-Reimer/undoclosetab) somehow does this it seems (as it is visible in the screens I attached in my first post). However this is not of great importance, just cosmetics.
